tests/manage/noobaa/test_basic_bucket_operations.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import pytest
+
+from tests.helpers import create_unique_resource_name
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.ocs.resources import noobaa
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
+    """
+    Returns a NooBaa resource that's connected to the S3 endpoint
+    Returns:
+        s3_res: A NooBaa resource
+
+    """
+    s3_res = noobaa.NooBaa()
+    return s3_res
+
+
+@pytest.fixture()
+def create_delete_buckets(request, noobaa_obj):
```

Comment:
we already have a test case for bucket creation.
Can we have this test case to verify bucket deletion? This means that the bucket creation should come in a setup fixture

Reply:
So bucket creation in setup fixture, and deletion as actual part of the test? I felt it would cover same scenario. I will change it.